sdk/ml/azure-ai-ml/azure/ai/ml/_schema/core/schema.py

```diff
@@ -12,14 +12,14 @@
 from marshmallow import fields, post_load, pre_load
 from pydash import objects
 
-from azure.ai.ml._schema.core.schema_meta import PatchedSchemaMeta
+from azure.ai.ml._schema.core.schema_meta import PatchedSchemaMeta, PatchedBaseSchema
 from azure.ai.ml._utils.utils import load_yaml
 from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY, FILE_PREFIX, PARAMS_OVERRIDE_KEY
 
 module_logger = logging.getLogger(__name__)
 
 
-class PathAwareSchema(metaclass=PatchedSchemaMeta):
+class PathAwareSchema(PatchedBaseSchema, metaclass=PatchedSchemaMeta):
```

Comment:
Why also directly inherited this?

Reply:
1. previous implementation will bring pylint error no-member in calling ComponentSchema().load()
2. it will inherit this class in PatchedSchemaMeta before so there will be no runtime impact
3. it enabled strong type intelligence for Schema objects in developing